Resources/config/services.yaml

```
@@ -3,7 +3,7 @@ parameters:
 
 services:
     ezstudio.installer.studio_installer:
-        class: %ezplatform.ee.installer.class%
+        class: '%ezplatform.ee.installer.class%'
```

Can we drop `ezplatform.ee.installer.class` parameter? 

```suggestion
        class: EzSystems\EzPlatformEnterpriseEditionInstallerBundle\Installer\Installer
```